<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuranWidget</class>
 <widget class="QWidget" name="QuranWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>733</width>
        <height>54</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="lableChapter">
         <property name="text">
          <string>Chapter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="line_Chapter">
         <property name="maximumSize">
          <size>
           <width>45</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lableVerse">
         <property name="text">
          <string>Verse</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="line_Verse">
         <property name="maximumSize">
          <size>
           <width>45</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lablePage">
         <property name="text">
          <string>Page</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="line_Page">
         <property name="maximumSize">
          <size>
           <width>45</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lableJuz">
         <property name="text">
          <string>Juz</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="label_Juz">
         <property name="maximumSize">
          <size>
           <width>45</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lableHizb">
         <property name="text">
          <string>Hizb</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="label_Hizb">
         <property name="maximumSize">
          <size>
           <width>45</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxTr">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxNoShowQuran">
         <property name="text">
          <string>No Show Quran</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="actionsLayout"/>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>85</width>
           <height>39</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="vLayout">
     <item>
      <widget class="QWebView" name="webView">
       <property name="contextMenuPolicy">
        <enum>Qt::ActionsContextMenu</enum>
       </property>
       <property name="url">
        <url>
         <string>about:blank</string>
        </url>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="playLayout"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="findLay"/>
   </item>
  </layout>
  <zorder>scrollArea</zorder>
  <zorder></zorder>
  <zorder></zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
